README.md

```diff
@@ -1,26 +1,48 @@
+| `limesurvey_smtp_password`                         |            v            | SMTP password                                                  |
+| `limesurvey_smtp_ssl`                              |   :heavy_check_mark:    | SMTP encryption either `ssl`, `tls` or ` `                     |
+| `limesurvey_smtp_debug`                            |           `0`           | SMTP debug option `0` or `1`                                   |
+| `limesurvey_admin_user`                            |         `admin`         | Admin nickname default                                         |
+| `limesurvey_admin_password`                        |   :heavy_check_mark:    | Admin password                                                 |
+| `limesurvey_admin_full_name`                       |       `Your Name`       | Full name of the administrator                                 |
+| `limesurvey_admin_email`                           | `your.name@example.com` | Admin email adress                                             |
+| `limesurvey_mail_admin_name`                       |       `Your Name`       | Sender name for outgoing emails                                |
+| `limesurvey_mail_admin_adress`                     | `your.name@example.com` | Sender adress of outgoing emails                               |
+| `limesurvey_mail_bounce_adress`                    | `your.name@example.com` | Email Adress for bounces                                       |
+| `limesurvey_ldap_plugin_server_url`                |   :heavy_check_mark:    | Url to your ldap server                                                 |
+| `limesurvey_ldap_plugin_server_port`               |          `389`          | Ldap port                                                 |
+| `limesurvey_ldap_plugin_ldap_version`              |   :heavy_check_mark:    | Ldap version either `2` or `3`'                                                 |
+| `limesurvey_ldap_plugin_opt_referrals`             |           `0`           | Select true if referrals must be followed (use false for ActiveDirectory)                                                |
+| `limesurvey_ldap_plugin_tls`                       |           `0`           | Use tls either `0` or `1`                                                |
+| `limesurvey_ldap_plugin_ldap_mode`                 |   :heavy_check_mark:    | How limesurvey should connect to ldap either `simplebind` or `searchandbind`                                                 |
+| `limesurvey_ldap_plugin_search_user_attribute`     |   :heavy_check_mark:    | Attribute to compare to the given login can be uid, cn, mail, ...                                                |
+| `limesurvey_ldap_plugin_user_search_base`          |   :heavy_check_mark:    | Base DN for the user search operation. Multiple bases may be separated by a semicolon (;)                                                 |
```

Comment:
Please implement a list for this. There is an ansible filter to combine a list with a user-defined delemiter.

Reply:
done